include/deal.II/lac/parallel_vector.h

```diff
@@ -140,6 +140,31 @@ namespace parallel
      * multiple threads. This may or may not be desired when working also with
      * MPI.
      *
+     * <h4>Limitations regarding the vector size</h4>
+     *
+     * This vector class is based on two different number types for indexing.
+     * The so-called global index type encodes the overall size of the vector.
+     * Its type is <code>types::global_dof_index</type>. The largest possible
```

Comment:
Mismatched markup. But I don't think you actually need it, doxygen should figure this out automatically